### PR TITLE
Added code level documentation and sample data for preview.

### PR DESCRIPTION
**Pull request by e-Garcia**
When a new class is created or a new feature is introduced, it's advisable to create some code level documentation to share how everything connects to each other and some helpful comments to help other developers understand what libraries or techniques were used.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,6 +1,22 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```
Comment (e-Garcia) [inline .idea/codeStyles/Project.xml:4]: We probably should remove some more .idea folder files. Let's explore which ones are required and which ones are not.

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="normal_margin">16dp</dimen>
+    <dimen name="margin_medium">16dp</dimen>
```
Comment (Joel-Ochoa) [inline app/src/main/res/values/dimens.xml:4]: we already have one margin for 16dp, I guess we should get rid of one of them

**Review by Joel-Ochoa — APPROVED** [review]

File: app/sampledata/emails [hunk 3]
```diff
@@ -0,0 +1,3 @@
+david@mail.com
```
Comment (e-Garcia) [inline app/sampledata/emails:1]: Here's an article about adding sample data for design preview: https://medium.com/androidiots/android-tools-attributes-and-sample-data-must-know-helpers-bf8477d3e144

File: app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (e-Garcia) [reply app/src/main/res/values/dimens.xml:3]: Thank you for pointing it out! :) Usually the size value goes after the margin or padding word so I updated the name to reflect that. Here's a sample of other open source project that uses that convention: https://github.com/android/views-widgets-samples/blob/master/CardView/Application/src/main/res/layout/fragment_card_view.xml

**Pull request merged**